docs-src/python/how-to-list-workflow-executions-using-the-client-in-python.md

```diff
@@ -0,0 +1,18 @@
+---
+id: how-to-list-workflow-executions-using-the-client-in-python
```

Comment:
I don't see https://github.com/temporalio/documentation/blob/96672716e70f49372fafa66bf1d58ba7ba0fea1a/docs/application-development/observability.md?plain=1#L686-L692 updated in this PR. Doesn't this need to be referenced in the guide?